packages/@react-spectrum/table/docs/TableView.mdx

```diff
@@ -560,6 +560,120 @@ function AsyncSortTable() {
 }
 ```
 
+## Column Resizing <VersionBadge version="beta" style={{marginLeft: 4, verticalAlign: 'bottom'}} />
+
+TableView supports resizable columns, allowing users to dynamically adjust the width of a column. To designate that a Column is resizable, provide it with the `allowsResizing` prop. This will render a draggable
+resizer handle that becomes visible on hover. Keyboard, touch, and screen reader users can start resizing by interacting with the target column's header and selecting the "Resize column" option
+from the dropdown.
+
+### Width values
+
+Columns support four different width props: `defaultWidth`, `width`, `minWidth`, and `maxWidth`. Each of these props accepts fixed values or percentages, with `width` and `defaultWidth` accepting fractional (`fr`) units as well.
+An initial, uncontrolled width can be provided to a Column using the `defaultWidth` prop. This allows the column width to freely shrink and grow in relation to other column widths. Alternatively, a controlled value can be provided
+by the `width` prop. Providing a non-fractional value `width` to a Column sets its width to the value provided to it, making it unaffected by other columns' widths. `minWidth` and `maxWidth` allows you to restrict a Column's
+width to a minimum and maximum value respectively. TableView prioritizes columns with defined widths and divides the remaining space evenly amongst the other columns.
+
+The example below illustrates how each of the aforementioned column width properties affects their respective column's resize behavior.
+
+```tsx example
+<TableView
+  aria-label="TableView with resizable columns"
+  width={800}
+  height={200} >
+  <TableHeader>
+    {/*- begin highlight -*/}
+    <Column key="file" allowsResizing maxWidth={500}>File Name</Column>
+    <Column key="type" allowsResizing defaultWidth="1fr">Type</Column>
+    <Column key="size" width={80}>Size</Column>
+    <Column key="date" allowsResizing minWidth={100}>Date Modified</Column>
+    {/*- end highlight -*/}
+  </TableHeader>
+  <TableBody>
+    <Row>
+      <Cell>2022-Roadmap-Proposal-Revision-012822-Copy(2)</Cell>
+      <Cell>PDF</Cell>
+      <Cell>214 KB</Cell>
+      <Cell>November 27, 2022 at 4:56PM</Cell>
+    </Row>
+    <Row>
+      <Cell>62259692_p0_master1200</Cell>
+      <Cell>PNG</Cell>
+      <Cell>120 KB</Cell>
+      <Cell>January 27, 2021 at 1:56AM</Cell>
+    </Row>
+  </TableBody>
+</TableView>
+```
+
+### Resize events
+
+TableView accepts an `onResize` prop which is triggered whenever a column resizer is moved by the user. This can be used in combination with the `width` prop to update a Column's width in a controlled fashion. TableView also accepts
+an `onResizeEnd` prop as well, triggered when the user finishes a column resize operation. Both events receive a Map object containing the widths of every column in the TableView.
```

Comment:
also...as well

```suggestion
TableView accepts an `onResize` prop which is triggered whenever a column resizer is moved by the user. This can be used in combination with the `width` prop to update a Column's width in a controlled fashion. TableView also accepts
an `onResizeEnd` prop, which is triggered when the user finishes a column resize operation. Both events receive a Map object containing the widths of every column in the TableView.
```